ui/components/component-library/text/README.mdx

```diff
@@ -0,0 +1,308 @@
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--text-align" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
+import { Text } from '../../ui/component-library/text';
+import { TEXT_ALIGN } from '../../../helpers/constants/design-system';
+
+<Text textAlign={TEXT_ALIGN.LEFT}>
+  left
+</Text>
+<Text textAlign={TEXT_ALIGN.CENTER}>
+  center
+</Text>
+<Text textAlign={TEXT_ALIGN.RIGHT}>
+  right
+</Text>
+<Text textAlign={TEXT_ALIGN.JUSTIFY}>
+  justify
+</Text>
+<Text textAlign={TEXT_ALIGN.END}>
+  end
+</Text>
+```
+
+### Overflow Wrap
+
+Use the `overflowWrap` prop and the `OVERFLOW_WRAP` object from `./ui/helpers/constants/design-system.js` to change the overflow wrap of the `Text` component
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--overflow-wrap" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
+import { Text } from '../../ui/component-library/text';
+import { OVERFLOW_WRAP } from '../../../helpers/constants/design-system';
+
+<div
+  style={{
+    width: 250,
+    border: '1px solid var(--color-error-default)',
+    display: 'block',
+  }}
+>
+  <Text overflowWrap={OVERFLOW_WRAP.NORMAL}>
+    {OVERFLOW_WRAP.NORMAL}: 0x39013f961c378f02c2b82a6e1d31e9812786fd9d
+  </Text>
+  <Text overflowWrap={OVERFLOW_WRAP.BREAK_WORD}>
+    {OVERFLOW_WRAP.BREAK_WORD}: 0x39013f961c378f02c2b82a6e1d31e9812786fd9d
+  </Text>
+</div>;
+```
+
+
+### Ellipsis
+
+Use the boolean `ellipsis` prop to change the if the `Text` component to have an ellipsis.
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--ellipsis" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
+import { Text } from '../../ui/component-library/text';
+
+<div
+  style={{
+    width: 250,
+    border: '1px solid var(--color-error-default)',
+    display: 'block',
+  }}
+>
+  <Text ellipsis>
+    Ellipsis: 0x39013f961c378f02c2b82a6e1d31e9812786fd9d
+  </Text>
+  <Text>
+    No Ellipsis: 0x39013f961c378f02c2b82a6e1d31e9812786fd9d
+  </Text>
+</div>;
+```
+
+### As
+
+Use the `as` prop to change the root html element of the `Text` component
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--as" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
+import { Text } from '../../ui/component-library/text';
+
+<Text as="dd">dd</Text>
+<Text as="div">div</Text>
+<Text as="dt">dt</Text>
+<Text as="em">em</Text>
+<Text as="h1">h1</Text>
+<Text as="h2">h2</Text>
+<Text as="h3">h3</Text>
+<Text as="h4">h4</Text>
+<Text as="h5">h5</Text>
+<Text as="h6">h6</Text>
+<Text as="li">li</Text>
+<Text as="p">p</Text>
+<Text as="span">span</Text>
+<Text as="strong">strong</Text>
+```
+
+Renders the html:
+
+```html
+<dd>dd</dd>
+
```

Comment:
nit: I think we can remove these line breaks in between the individual html elements?